debug/index.html

```diff
@@ -42,113 +42,116 @@
 		<div id="trace">Rendering...</div>
 	</div>
 	<script type="text/javascript">
-        var tilesUrl = 'http://osmosnimki.ru/vtile/{z}/{x}/{y}.js';
+        $( document ).ready(function() {
+
+            //var tilesUrl = 'http://osmosnimki.ru/vtile/{z}/{x}/{y}.js';
+            var tilesUrl = 'test-tile.js?&x={x}&y={y}&z={z}';
```

Comment:
Should this change persist? Looks like debug code. If yes, please just remove the old code.

Reply:
I did it because the Webserver of http://osmosnimki.ru is not responding anymore. So i did add the Tile directly to have a working example again. I guess it should persist like this.